src/app/Models/employee.model.ts

```diff
@@ -1,7 +1,13 @@
 export interface Employee {
-  status: string
+  status: number,     // our status is a number which will eventually switch to a enum or be referenced from a database
   id: string,
-  name: string;
-  email: string;
-  phone: number;
+  name: string,
+  email: string,
+  phone: string      // phone numbers are not numbers pal they are a string which only contains numbers
 }
+
+export type EmployeeStatus = {
```

Comment:
this new type is what drives things, basically the CDK thing you can pass data with the DropListData, this type defines the data I am pushing down